docs/documentation/features/providedServices.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: documentation
+---
+
+# Provided Services
+
+In this chapter useful services of the Eclipse SmartHome project are described. 
+
+## Obtaining the default IP address
+
+Some things might require a `callback` URL which should be bound to a certain network interface. A user can configure his default network address via Paper UI under `Configuration -> System -> Network Settings`. To obtain this configured address the `ThingHandlerFactory` needs a `service reference` to the `NetworkInterfaceService` in its `MyHandlerFactory.java`:
```

Comment:
Do not talk about "things" and "handlers", we are not in the binding docs anymore. Rather describe your service in general and merely mention in a side note that it might be helpful for callback functionality in ThingHandlers and AudioSinks.